Assets/Scripts/DoorController.cs

```diff
@@ -8,6 +8,9 @@ public class DoorController : InteractableItem
     private bool _open = false;
     private bool _broken = true;
     public BoxCollider2D physicalCollider;
+    public BoxCollider2D standCollider;
+    public Sprite UseSprite;
```

Comment:
Possibly take the sprite from parent? Not sure here, we may want to change the sprite for specific action. Or not.